torchvision/prototype/transforms/_misc.py

```diff
@@ -104,6 +104,11 @@ def __init__(self, mean: Sequence[float], std: Sequence[float]):
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return F.normalize(inpt, mean=self.mean, std=self.std)
 
+    def forward(self, *inpts: Any) -> Any:
+        if has_any(inpts, PIL.Image.Image):
+            raise TypeError(f"{type(self).__name__}() does not support PIL images.")
+        return super().forward(*inpts)
```

Comment:
Adopting @vfdev-5's proposal to eliminate the `sample = inputs if len(inputs) > 1 else inputs[0]` idiom in favour of `super().forward(*inpts)`